### PR TITLE
Properly style admonitions from Sphinx

### PR DESCRIPTION
**Pull request by borft**
- refactor sphinx admonitions so they work with docusaurus styling
- make external linkns open in new tab/window

### CODE REVIEW

File: docs/src/components/sphinx-page/index.tsx [hunk 1]
```diff
@@ -32,17 +32,90 @@ const SphinxPage = (props) => {
         's1': {'class': 'token string', 'style': 'color: rgb(227, 17, 108);'}                      // string
     };
 
+    const admonitionMap = {
+        'admonition attention': {
+            'class': 'admonition admonition-caution alert alert--warning',
+            'icon': '<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><path fill-rule="evenodd" d="M8.893 1.5c-.183-.31-.52-.5-.887-.5s-.703.19-.886.5L.138 13.499a.98.98 0 0 0 0 1.001c.193.31.53.501.886.501h13.964c.367 0 .704-.19.877-.5a1.03 1.03 0 0 0 .01-1.002L8.893 1.5zm.133 11.497H6.987v-2.003h2.039v2.003zm0-3.004H6.987V5.987h2.039v4.006z"/></svg>'
```
Comment (borft) [inline docs/src/components/sphinx-page/index.tsx:38]: @sdirosa for now I've just added a list of all icons

File: docs/src/components/sphinx-page/index.tsx [hunk 2]
```diff
@@ -32,17 +32,90 @@ const SphinxPage = (props) => {
...
             })
-          .then(raw => {
+            .then(raw => {
                 const tempDiv = document.createElement('div');
                 tempDiv.innerHTML = raw;
 
+                // fix admonitions
+                // <p class="admonition-title"> ==> <div class="admonition-heading"><h5>[TITLE]</h5></div>
+                // <div class="admonition note"> ==> <div class="admonition admonition-tip alert alert--success">
+                // <div class="admonition warning"> ==> <div class="admonition admonition-danger alert alert--danger">
```
Comment (ivarpruijn) [inline docs/src/components/sphinx-page/index.tsx:92]: Remove, or update with the icon example as well?

Comment (borft) [reply docs/src/components/sphinx-page/index.tsx:92]: sure; done!

**Review by ivarpruijn — APPROVED** [review]
Nice solution!

File: docs/src/components/sphinx-page/index.tsx (hunk 1 above)
Comment (sdirosa) [reply docs/src/components/sphinx-page/index.tsx:40]: I think this works alright for now 👌🏼

**Pull request merged**